src/ray/raylet/node_manager.cc

```diff
@@ -809,6 +809,9 @@ void NodeManager::AssignTask(Task &task) {
     lineage_cache_.AddReadyTask(task);
     // Mark the task as running.
     local_queues_.QueueRunningTasks(std::vector<Task>({task}));
+    // Notify the task dependency manager that we no longer need this task's
+    // object dependencies.
+    task_dependency_manager_.UnsubscribeDependencies(task_id);
```

Comment:
Where is `task_id` defined?

Reply:
Oops, thanks! Guess I should try compiling before I push :P